home/tests.py

```diff
@@ -1,3 +1,35 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.fixture
+def profile():
```

Comment:
Considering that adding the user and profile records is a complex operation, are you sure you don't need a model method for it?

Also consider that you probably want to be adding the two DB object in a single transaction